app/views/profiles/search.html.erb

```diff
@@ -0,0 +1,98 @@
+<div class = "search-page" id="friend-lookup">
+  
+  <div class = "col-md-6 col-md-offset-3">
+  	<div class="table-responsive">
+		  <table class="table search-table" >
+		    <thead>
+		    	<tr>
+		    		<th class="header search-header">Summoner Search Result</th>
+		    	</tr>
+		    </thead>
+		    <tbody>
```

Comment:
There is little advantage to using a table if you only make one big row with one data element.  You should use 3 rows  in the table body with three `<td>` each:
Row1: Summoner Name, Most Played Champ, icon
Row2: Summoner Level, Highest KDR, icon
Row3: Last Active, Highest WLR, icon

The icon takes up the entire last column using a `colspan` tag.